node/src/components/gossiper/config.rs

```diff
@@ -18,12 +18,15 @@ pub(super) const MAX_SATURATION_LIMIT_PERCENT: u8 = 99;
 pub(super) const DEFAULT_FINISHED_ENTRY_DURATION: &str = "60sec";
 const DEFAULT_GOSSIP_REQUEST_TIMEOUT: &str = "10sec";
 const DEFAULT_GET_REMAINDER_TIMEOUT: &str = "60sec";
+const DEFAULT_VALIDATE_AND_STORE_TIMEOUT: &str = "60sec";
 #[cfg(test)]
 const SMALL_TIMEOUTS_FINISHED_ENTRY_DURATION: &str = "2sec";
 #[cfg(test)]
 const SMALL_TIMEOUTS_GOSSIP_REQUEST_TIMEOUT: &str = "1sec";
 #[cfg(test)]
 const SMALL_TIMEOUTS_GET_REMAINDER_TIMEOUT: &str = "1sec";
+#[cfg(test)]
+const SMALL_TIMEOUTS_VALIDATE_AND_STORE_TIMEOUT: &str = "1sec";
```

Comment:
This seems rather optimistic. What if event queues are swamped? Even for addresses, which don't need validation, we might not get there in 1 sec. Then the data is stored eventually but not gossiped onwards, right?